sdk/kiosk/src/tx/kiosk.ts

```diff
@@ -200,20 +203,16 @@ export function purchase(
   tx: TransactionBlock,
   itemType: string,
```

Comment:
Some argument grouping / ordering would help. Eg kioskId, itemId, itemType?

Reply:
I was trying to follow the same ordering we had on the other ones. (itemType was always first)